contracts/manager/Manager.sol

```diff
@@ -186,40 +138,40 @@ contract Manager is IManager, Actor, ManagerBase {
     return getActors(SAFE_RECIPIENT);
   }
 
-  // internal functions
-
   // @dev Validates the request.
```

Comment:
Doc block not updated to new parameters.

Reply:
I know. I was too tired when I finished yesterday and left it for next PR, when I will fix the coverage